v2/internal/testcommon/vcr/redact.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package vcr
+
+import (
+	"net/http"
+	"regexp"
+)
+
+var RequestHeadersToRemove = []string{
```

Comment:
minor: Do these need to be exported? (and below w/ `ResponseHeadersToRemove`)?

It means external callers could mutate them which seems ick. AFAIK we don't want/need them to be externally configurable as they're an implementation detail of the `vcr` package? Exporting `RedactRequestHeaders` is enough?